src/core/state_test.go

```diff
@@ -59,7 +59,8 @@ func TestExpandVisibleOriginalTargets(t *testing.T) {
 
 func TestExpandOriginalSubLabels(t *testing.T) {
 	state := NewDefaultBuildState()
-	state.AddOriginalTarget(BuildLabel{PackageName: "src/core", Name: "..."}, true)
```

Comment:
I don't think we ever actually needed to support this. The new implementation just panics but things like `plz test //...` are still fine.